client/state/db_bolt.go

```diff
@@ -140,6 +140,12 @@ var (
 	nodeRegistrationKey = []byte("node_registration")
 
 	hostVolBucket = []byte("host_volumes_to_create")
+
+	// clientIdentityBucket and clientIdentityBucketStateKey are used to persist
+	// the client identity and its state. Each client will only have a single
+	// identity, so we use a single key value for the storage.
+	clientIdentityBucket         = []byte("client_identity")
+	clientIdentityBucketStateKey = []byte("client_identity_state")
```

Comment:
_Super_ nitpicky: do we want to refer to these as "node identity" to match everywhere else?